roles/login-override/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+  - name: Get checksum for Galaxy Client Login.vue
+    shell: "md5sum {{ galaxy_root }}/client/src/components/login/Login.vue | awk '{print $1;}'"
+    register: galaxy_login_checksum
```

Comment:
We could alternatively use the ansible patch module: https://docs.ansible.com/ansible/latest/collections/ansible/posix/patch_module.html and apply a git diff. If the patch fails to apply cleanly, it will be detectable as an incompatible change. As long as we know which version the patch last applied cleanly to, it should also be possible to see the code in full context if necessary? This is assuming the changes are fairly incremental. If the changes are substantive, this checksum approach makes more sense?